tests/ui/unit/test-setup.js

```diff
@@ -1,5 +1,6 @@
 // Entry point for Jest tests
 import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
+import 'jest-dom/extend-expect';
```

Comment:
This library adds additional functionality such as the `toBeInTheDocument` and `toHaveClass`. Importing only functions you need means you need to do `expect.extend({toBeInTheDocument, toHaveClass})` for every [usage](https://www.npmjs.com/package/jest-dom#usage).